src/main/java/io/lettuce/core/api/async/RedisScriptingAsyncCommands.java

```diff
@@ -85,6 +85,19 @@ public interface RedisScriptingAsyncCommands<K, V> {
      */
     <T> RedisFuture<T> eval(byte[] script, ScriptOutputType type, K[] keys, V... values);
 
+    /**
+     * This is a read-only variant of the EVAL command that cannot execute commands that modify data.
+     *
+     * @param script Lua 5.1 script.
+     * @param type the type.
+     * @param keys the keys.
+     * @param values the values.
+     * @param <T> expected return type.
+     * @return script result.
+     * @since 6.4
```

Comment:
Could we change this to `@since 7.0`

We are preparing a the 7.0 release right now and I think we will not benefit from a minor release right now

Reply:
Sure, let me fix that real quick.